zebra/if_netlink.c

```diff
@@ -365,14 +365,16 @@ static void netlink_vrf_change(struct nlmsghdr *h, struct rtattr *tb,
 	}
 }
 
-static int get_iflink_speed(struct interface *interface)
+static int get_iflink_speed(struct interface *interface, int *error)
```

Comment:
can this be `enum` to catch the more readable code in the future? or there will be only 0 and -1? :)

Reply:
up to now,I can even use a boolean.